### PR TITLE
Font-family declarations fixed

### PR DESCRIPTION
**Pull request by uyouthe**
My PR is a:
🐛 Bug fix

Main update on the:
Overall project styling

I've encountered the wrong font-family declaration in admin panel. It was
```CSS
font-family: Lato-Bold;
```
while it should be
```CSS
font-family: 'Lato', sans-serif;
font-weight: bold;
```

The old declaration resulted in headlines were rendered in Times New Roman. You should always provide `sans-serif` or other font type fallback, and you also should never declare it "Bold" or "Medium" right in `font-family`.

Guys, I find exactly 240 font-family declarations across the project. You use SCSS anyway, why not establish a mixin system?

You also got double and single quotes messed up. You _really_ need a linter. But I love Strapi and just want to help! Please, merge it.

### CODE REVIEW

File: packages/strapi-admin/admin/src/components/Sub/styles.scss [hunk 1]
```diff
@@ -10,7 +10,8 @@
 
   > span {
     text-decoration: none;
-    font-family: Lato-Bold;
+    font-family: 'Lato', sans-serif;
+font-weight: bold;
```
Comment (maturanomx) [inline packages/strapi-admin/admin/src/components/Sub/styles.scss]: Bad indentation

File: packages/strapi-admin/admin/src/containers/HomePage/styles.scss [hunk 2]
```diff
@@ -9,13 +9,14 @@
   line-height: 18px;
   text-decoration: none;
   > span {
-    font-family: Lato-Bold;
+    font-family: 'Lato', sans-serif;
+font-weight: bold;
```
Comment (maturanomx) [inline packages/strapi-admin/admin/src/containers/HomePage/styles.scss]: Same. Bad indentation.

**Comment by uyouthe** [comment]
Guys?...

**Comment by uyouthe** [comment]
So?... merge or not?

**Comment by maturanomx** [comment]
patience

**Comment by uyouthe** [comment]
Yeah! Thank you guys!

**Pull request merged**